packages/eslint-config-expo/README.md

```diff
@@ -0,0 +1,35 @@
+# eslint-config-expo
+
+Base ESLint config for Expo projects.
+
+## Installation
+
+```sh
+yarn add --dev eslint-config-expo
+```
+
+You will also need to install `eslint`:
+
+```sh
+yarn add --dev eslint
+```
+
+## Usage
+
+Import this config into your own ESLint configuration using the `extends` option. ESLint checks both `package.json` and `.eslintrc.*` files for its configuration:
+
+### package.json
+```js
+{
+  "eslintConfig": {
+    "extends": "expo"
```

Comment:
What do you think about mentioning `eslint:recommended` here?
```suggestion
    "extends": ["expo", "eslint:recommended"]
```

Reply:
Good idea!